security/index.html

```diff
@@ -6,25 +6,29 @@
 ---                      
 
         
-        <!-- end Security -->
+        <!-- begin Security -->
         <hr class="whitespace">
         <div id="secvuln-reporting" class="row">
-            <p>See our <a href="{{ site.baseurl }}/security/advisories/">archive of past security advisories</a>.</p>
-            <div class="small-12">
-                <div class="callout primary">
-                    <p>If you discover a security vulnerability or would like to report a security issue privately and securely, please <a href="mailto:security@openmicroscopy.org.uk">email us</a>. You can use GPG keys to communicate with us securely. If you do, please upload your GPG public key or supply it to us in some other way, so that we can reply securely too:</p>
+            <div class="column small-12">
+                <h2>Security Advisories</h2>
+                <p>See our archive of <a href="{{ site.baseurl }}/security/advisories/">past security advisories</a>.</p>                
+                <h2>How to Report a Security Vulnerability</h2>
+                <p>If you discover a security vulnerability or would like to report a security issue privately and securely, please <a href="mailto:security@openmicroscopy.org.uk">email us</a>. You can use GPG keys to communicate with us securely. If you do, please upload your GPG public key or supply it to us in some other way, so that we can reply securely too:</p>
```

Comment:
`security@openmicroscopy.org` should now be an alias of `security@openmicroscopy.org.uk`. I propose we update the `mailto` href to use the former /cc @kennethgillen @joshmoore